setup.py

```diff
@@ -9,7 +9,14 @@ def read(fname):
 
 
 extras = {
-    "standard": ["pandas", "rtree", "pyyaml", "snakemake", "sphinx-rtd-theme>=0.5.1"],
+    "standard": [
+        "pandas",
+        "rtree",
+        "pyyaml",
+        "snakemake",
+        "sphinx-rtd-theme>=0.5.1",
+        "pandoc",
```

Comment:
Does it make sense to add it here even if it cannot be installed via pip?

Reply:
you are right! I added this in the first place. Will remove it